peg-macros/translate.rs

```diff
@@ -168,8 +168,15 @@ fn make_parse_state(grammar: &Grammar) -> TokenStream {
         }
     }
 
+    let trace = if cfg!(feature = "trace") {
+        quote!( #[derive(Debug)] )
+    } else {
+        quote!()
+    };
+
     quote_spanned! { span =>
         #[allow(unused_parens)]
+        #trace
```

Comment:
I think, you can just always derive `Debug`. Unused trait impls will be eliminated by rustc

Reply:
Hmm no. I've decided to skip all the fields now. There is no need for `Debug` in ParseState anymore.